src/Avalonia.Base/Platform/VisualQueryProvider.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Avalonia.Platform
+{
+    internal class VisualQueryProvider
+    {
+        private readonly Visual _visual;
+
+        public double Width { get; private set; } = double.PositiveInfinity;
+
+        public double Height { get; private set; } = double.PositiveInfinity;
+
+        public VisualQueryProvider(Visual visual)
+        {
+            _visual = visual;
+        }
+
+        public event EventHandler? WidthChanged;
+        public event EventHandler? HeightChanged;
+
+
+/* Unmerged change from project 'Avalonia.Base (netstandard2.0)'
```

Comment:
There are several of these merge comment in various files from this commit; they should be removed.

Reply:
Removed